src/Psalm/Internal/Type/Comparator/ObjectComparator.php

```diff
@@ -134,7 +137,7 @@ public static function isShallowlyContainedBy(
 
     /**
      * @param  TNamedObject|TTemplateParam|TIterable  $type_part
-     * @return array<string, TNamedObject|TTemplateParam|TIterable|TObjectWithProperties>
+     * @return array<string, TNamedObject|TTemplateParam|TIterable|TObjectWithProperties|TCallableObject>
```

Comment:
Not 100% sure about this, as this might be a BC break.

Reply:
Changes in internal classes (like this) are not considered BC breaks.